src/pages/WorkflowInputs.js

```diff
@@ -34,19 +25,7 @@ export const WorkflowInputs = ({ workflowUrl, workflowInputs, setWorkflowInputs,
             onChange: setWorkflowInputs
           })
         ])]),
-        div({ style: { display: 'flex', marginTop: '1rem', justifyContent: 'space-between' } }, [
-          'Outputs will be saved to cloud storage',
-          div([
-            h(ButtonOutline, {
-              onClick: displayWorkflowUrlPage
-            }, ['Back to previous page']),
-            h(ButtonPrimary, {
-              style: { marginLeft: '1rem' },
-              disabled: !workflowInputs,
-              onClick: () => submitRun()
-            }, ['Run workflow'])
-          ])
-        ])
+        div({ style: { marginTop: '1rem' } }, ['Outputs will be saved to cloud storage'])
```

Comment:
I would put this line in the `SubmitWorkflow` component too (it's more to do with submitting the workflow than choosing its inputs)

Reply:
Hmmm, I see what you mean. I will move it to `SubmitWorkflow`